### DISCUSSION TITLE
Github Copilot and Microsoft 365 Copilot question

### DISCUSSION THREAD

**Discussion by MalditoBarbudo** [community/community#77305] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

If my work has the Microsoft 365 Copilot addon active, do I have access also to GitHub copilot if I link my work email to my GitHub account?

I tried to find info about this but didn't find anything, so I just thought of asking here.

**Comment by Akash1134** [comment, marked as answer]
👋 Hi @MalditoBarbudo ,

Interesting question!

However, GitHub Copilot requires separate subscription for access. For details, please checkout the [features and pricing](https://github.com/features/copilot) page. 

Thanks!

**Reply by MalditoBarbudo** [reply]
@Akash1134 thanks for the quick answer.
